Gemfile

```diff
@@ -14,6 +14,9 @@ gem 'fast_jsonapi'
 gem 'capybara'
 gem 'faraday'
 gem 'figaro'
+gem 'bcrypt'
+gem 'jwt'
```

Comment:
allows for calling Json Web Token to assist in creating user api_key. JWT is called for encoding which creates the api key and is called again for decoding wich decodes the api key to authenticate a user when logging in